radixconfig.yaml

```diff
@@ -58,7 +58,7 @@ spec:
       publicPort: flask
 
     - name: dmss
-      image: mariner.azurecr.io/dmss:latest
+      image: mariner.azurecr.io/dmss:v0.3.14
```

Comment:
maybe use DMSS v0.3.14 tag in docker-compose also?
(the local dev containers from docker-compose up will probably stop working if breaking changes are made to dmss in the future)